go.mod

```diff
@@ -397,7 +397,7 @@ replace (
 
 	// Our cosmos-sdk branch is: https://github.com/osmosis-labs/cosmos-sdk/tree/osmo-v23/v0.47.5-iavl-v1, current branch: osmo-v23/v0.47.5-iavl-v1. Direct commit link: https://github.com/osmosis-labs/cosmos-sdk/commit/d44a3847e089de47cbba0f31a0cebed64ad2806f
 	// https://github.com/osmosis-labs/cosmos-sdk/releases/tag/v0.47.5-v23-osmo-1
-	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.47.5-v23-osmo-2-iavl-v1
+	github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.47.6-0.20240227001520-a836ab6f5815
```

Comment:
Do we not need to bump it to this commit? https://github.com/osmosis-labs/cosmos-sdk/commit/10406e3b6d6ed3632d66847a08e8480c4e81015a

Reply:
That is the next PR, where I bump prometheus here in osmosis. I wanted to separate the changes

Reply:
The PR is open but in draft, it is what alerted me to the fact that osmosis was behind and failing tests

Reply:
gotcha. Thanks for the explanation!